server/cmwell-util/src/main/scala/cmwell/util/http/HttpZipDecoder.scala

```diff
@@ -0,0 +1,21 @@
+package cmwell.util.http
+
+import akka.http.scaladsl.coding.{Deflate, Gzip, NoCoding}
+import akka.http.scaladsl.model.HttpResponse
+import akka.http.scaladsl.model.headers.HttpEncodings
+
+object HttpZipDecoder {
```

Comment:
It's not so much a "http util" as a "akka util".
Lets move it to `cmwell.util.akka.http` package

WDYT?

Reply:
I'm happy @hochgi , good suggestion. Is there anything else?

Reply:
@hochgi that's done now...